elevenlabs/api/voice.py

```diff
@@ -146,6 +172,11 @@ def from_api(cls, api_key: Optional[str] = None):
         url = f"{api_base_url_v1}/voices"
         response = API.get(url).json()
         return cls(**response)
+    
+    @classmethod
+    def default_settings(cls):
```

Comment:
Rethinking about this maybe it's better to put it into the `Voice` object? As `Voice.default_settings()`

Reply:
Fair enough, will do that soon.